### DISCUSSION TITLE
How to use snap_dialog?

### DISCUSSION THREAD

**Discussion by ohyung1287** [MetaMask/snaps#973] [Q&A] [answered]
I got a warning that says "snap_confirm is deprecated. Use snap_dialog instead."
But when I try to replace snap_confirm with snap_dialog, the error `Problem happened: The method "snap_dialog" does not exist / is not available` showed up.
I do see snap_dialog inside metamask/rpc-methods. How do I import it to use snap_dialog?
Thanks

**Comment by andyv09** [comment, marked as answer]
Probably not yet available in the latest available Flask. You might solve this issue by building the [develop branch of MetaMask Extension](https://github.com/MetaMask/metamask-extension) and using that extension instead.

**Reply by ritave** [reply]
I confirm that this deprecation information might have been added too soon. The next release of Flask will have `snap_dialog` included.

We'll maintain `snap_confirm` in that above mentioned release, and remove it in the following one.
